_episodes/02-filedir.md

```diff
@@ -194,7 +194,7 @@ $ ls -F /
 `ls` is the **command**, with an **option** `-F` and an
 **argument** `/`.
 We've already encountered options (also called **switches** or **flags**) which
-either start with a single dash (`-`) or two dashes (`--`), and they change the behaviour of a command.
```

Comment:
Not sure this change is needed. Need to check the style guide.

Reply:
I'm pretty sure being an american organization SWC style guide mandates american english spellings.